editor/project_settings_editor.cpp

```diff
@@ -965,8 +965,6 @@ void ProjectSettingsEditor::_action_add() {
 	while (r->get_next())
 		r = r->get_next();
 
-	if (!r)
-		return;
```

Comment:
I have to wonder why this was removed. It seems like it'd be needed.

Reply:
I don't think so.
The while loop move to last element. Since it is last element it can't be null.
One exception is when r is already null. But that too is covered by another `if(!r)` present before the loop.